docs/adrs/0005-processed-dois.md

```diff
@@ -0,0 +1,24 @@
+# 5. Storing processed DOIs to prevent duplicate items
+
+## Status
+
+Proposed
+
+## Context
+
+Given that Wiley will not provide a monthly list of only new DOIs and will instead just update a master list of DOIs, we need a system to ensure that we do not deposit the same article twice.
+
+## Decision
+
+We will create a list of processed DOIs that the automated workflow will check against while processing the updated master list each month. The steps of the process are outlined below:
```

Comment:
"By processed DOIs it is meant that the DOI has been successfully published in DSpace."